### DISCUSSION TITLE
Mods name request

### DISCUSSION THREAD

**Discussion by doemty** [UltimatChamp/optimum-fuse#21] [Q&A] [answered]
![2024-05-26_22 57 05](https://github.com/UltimatChamp/optimum-fuse/assets/151661389/c0861d89-292f-4760-8378-a1ae0b4b1a01)
![2024-05-26_22 57 07](https://github.com/UltimatChamp/optimum-fuse/assets/151661389/be1bee62-143a-462d-98f9-20be9800402a)
![2024-05-26_23 00 02](https://github.com/UltimatChamp/optimum-fuse/assets/151661389/7749fd47-aaf2-43cd-92de-b8792eebaa64)

1. Which mod adds description for food?
2. Which mods adds description for enchantments?
3. Which mods are affecting on armor? I mean view chestplate on stand, icon, and white tooltip

srry for so many questions >.<

**Comment by UltimatChamp** [comment, marked as answer]
1. Immersive:Tooltip
2. It looks like the enchantment descriptions are getting duplicated😅. You can remove **Enchantment Descriptions**. The description is already being given by Immersive:Tooltip.
3. Immersive:Tooltip

I'll like to know your opinion on Immersive:Tooltip, since, it makes every Tooltip look good, but can be distracting.

**Reply by doemty** [reply]
thx <3

**Reply by doemty** [reply]
Oh, I have just noticed your question.
I like this mod, but honestly I do not like it's enchantment description(it's too bright compared to **Enchantment Descriptions**).
![image](https://github.com/UltimatChamp/optimum-fuse/assets/151661389/682fe2ac-aa7c-4d0e-adaa-3ab5471be045)
Also it has some incompatibilities with **Quark** (dublicates effects, but do not show levels of effects)
If you are looking for analog, I can recommend **Obscure Tooltips** or **Legendary Tooltips.**

**Reply by UltimatChamp** [reply]
thx, I've requested the author of the mod to add a config system to disable or tweak some features like color of tooltips!

**Reply by UltimatChamp** [reply]
It looks like the golden enchantment description was done by some other mod that you have included on top of the modpack.

**Reply by UltimatChamp** [reply]
Immersive:Tooltip doesnt do that🤔

**Reply by doemty** [reply]
> Immersive:Tooltip doesnt do that🤔

Wait, I will check it now

**Reply by doemty** [reply]
> Immersive:Tooltip doesnt do that🤔

Oh, you are right. But honestly I do not know what mod adds it  :/
